frontend/src/components/settings/ManageUserDialog.vue

```diff
@@ -175,19 +177,16 @@ export default {
         this.isDisplayed = value
       },
     },
-    user: {
-      handler() {
-        this.userOperationType = Object.keys(this.user).length === 0 ? 'adding' : 'updating';
+    updatingUser: {
+      handler(value) {
+        this.userOperationType = Object.keys(value).length === 0 ? 'adding' : 'updating';
```

Comment:
I might be inclined to have the operation type as a prop rather than use a rule of thumb to determine the operation (e.g. if you wanted to default a value in the user that's passed in it wouldn't be empty). But I don't think it's a necessary change for now.

Reply:
Understood. Left for now.